Importing modules and data

In [8]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import plotly.express as px
from collections import Counter

In [9]:
original_data=pd.read_csv('mkbhd_cleaned.csv')
df=original_data.copy()
df

Unnamed: 0                                           Title    Views  \
0              0                       YouTube Needs to Fix This   435998   
1              1              The Electric Car Pre-Order Problem  1492800   
2              2  Does Fast Charging ACTUALLY Ruin Your Battery?  2923457   
3              3        "The Apple Studio Display is a Bad Deal"  3228172   
4              4              Mac Studio Review: Double Trouble!  3022613   
...          ...                                             ...      ...   
1408        1423                 Fraps HD Test in 1080p (18 WOS)   130044   
1409        1424   HP Pavilion dv7t Media Center Remote Overview  4640300   
1410        1425                     High fps LG Voyager footage   179533   
1411        1426           14 Year knock-down shot (11 Handicap)   175876   
1412        1427                 13-Year-Old Golf Swing Analysis   781563   

            Date   Likes  Comments                             Link  
0     2022-04-01   52217      4919  youtube.com/watch?v=1Cw-vODp-8Y  
1     2022-03-30   83763      7958  youtube.com/watch?v=a6zvvlrd-jw  
2     2022-03-25  174602     12325  youtube.com/watch?v=UpqaQR4ikig  
3     2022-03-22  124575     11698  youtube.com/watch?v=yoigsHYc77s  
4     2022-03-18  120729      6390  youtube.com/watch?v=wvhP1al3Ur4  
...          ...     ...       ...                              ...  
1408  2009-01-01    1592       237  youtube.com/watch?v=hCEBLSSU-2Y  
1409  2009-01-01  234631     37125  youtube.com/watch?v=9gk_rl3y_SU  
1410  2008-09-15    2644       724  youtube.com/watch?v=f7mqssf66l4  
1411  2008-05-30    2198       358  youtube.com/watch?v=PD5igzFB8iY  
1412  2008-03-28   17546      3545  youtube.com/watch?v=fjhmnWFsr3s  

[1413 rows x 7 columns]

In [10]:
df.dtypes

Unnamed: 0     int64
Title         object
Views          int64
Date          object
Likes          int64
Comments       int64
Link          object
dtype: object

Lets remove Unnamed column and fix the dtyep of Date column

In [11]:
df.drop('Unnamed: 0',axis=1,inplace=True)
df['Date']=pd.to_datetime(df['Date'])

In [12]:
df.head()

Title    Views       Date   Likes  \
0                       YouTube Needs to Fix This   435998 2022-04-01   52217   
1              The Electric Car Pre-Order Problem  1492800 2022-03-30   83763   
2  Does Fast Charging ACTUALLY Ruin Your Battery?  2923457 2022-03-25  174602   
3        "The Apple Studio Display is a Bad Deal"  3228172 2022-03-22  124575   
4              Mac Studio Review: Double Trouble!  3022613 2022-03-18  120729   

   Comments                             Link  
0      4919  youtube.com/watch?v=1Cw-vODp-8Y  
1      7958  youtube.com/watch?v=a6zvvlrd-jw  
2     12325  youtube.com/watch?v=UpqaQR4ikig  
3     11698  youtube.com/watch?v=yoigsHYc77s  
4      6390  youtube.com/watch?v=wvhP1al3Ur4

In [13]:
df.dtypes

Title               object
Views                int64
Date        datetime64[ns]
Likes                int64
Comments             int64
Link                object
dtype: object

# EDA

### Lets focus on date column

In [14]:
df['Date'].dt.year.value_counts().sort_index()

2008      3
2009    279
2010     70
2011     56
2012     99
2013     97
2014     93
2015     87
2016     83
2017    100
2018     99
2019    110
2020    121
2021     94
2022     22
Name: Date, dtype: int64

In [15]:
px.bar(df['Date'].dt.year.value_counts().sort_index(),title='Count of videos in each year from 2008-2022')

He released a lot of videos in 2009. Lets see his release pattern in that year.

In [16]:
df[df['Date'].dt.year==2009]['Date'].dt.month.value_counts().sort_index()

1     22
2     93
3     35
4     25
5     20
6     22
7     13
8     13
9      4
10    14
11     9
12     9
Name: Date, dtype: int64

In [17]:
px.bar(df[df['Date'].dt.year==2009]['Date'].dt.month.value_counts().sort_index(),title='Count of videos in each month of the year 2009')

He released a lot of videos in feb 2009. Lets go another step further and focus on that particular month of 93 videos

In [18]:
df[(df['Date'].dt.year==2009) & (df['Date'].dt.month==2)]['Date'].dt.day.value_counts().sort_index()

1     5
2     8
3     4
4     6
6     2
7     6
8     8
9     5
10    4
11    3
12    6
13    3
14    1
15    2
16    4
17    2
18    3
19    4
20    2
21    3
22    4
23    4
24    1
25    1
26    1
27    1
Name: Date, dtype: int64

In [19]:
px.bar(df[(df['Date'].dt.year==2009) & (df['Date'].dt.month==2)]['Date'].dt.day.value_counts().sort_index())

One thing we can observe is, he uploaded more videos in the start of the month when compared to end of the month.

Lets get out of 2009 year.

Lets count number of videos he released in each month of every year

In [20]:
df['Date'].dt.month.value_counts().sort_index()

1     129
2     185
3     123
4     116
5      97
6     110
7      85
8     106
9     106
10    127
11    111
12    118
Name: Date, dtype: int64

In [21]:
px.bar(df['Date'].dt.month.value_counts().sort_index())

From above we observe that some more videos were released in feb. I think they are coming from 2009 year. Lets exclude 2009 year and do this again

In [22]:
df[df['Date'].dt.year!=2009]['Date'].dt.month.value_counts().sort_index()

1     107
2      92
3      88
4      91
5      77
6      88
7      72
8      93
9     102
10    113
11    102
12    109
Name: Date, dtype: int64

In [23]:
px.bar(df[df['Date'].dt.year!=2009]['Date'].dt.month.value_counts().sort_index())

Now this seems more consistent

Lets do the same for days

In [24]:
df['Date'].dt.day.value_counts().sort_index()

1     48
2     56
3     46
4     46
5     36
6     36
7     48
8     43
9     55
10    38
11    53
12    52
13    49
14    44
15    46
16    54
17    46
18    56
19    47
20    44
21    42
22    48
23    55
24    40
25    41
26    48
27    38
28    42
29    47
30    39
31    30
Name: Date, dtype: int64

In [25]:
px.bar(df['Date'].dt.day.value_counts().sort_index())

There is no pattern here

## Lets focus on Likes column

In [26]:
df['Likes']

0        52217
1        83763
2       174602
3       124575
4       120729
         ...  
1408      1592
1409    234631
1410      2644
1411      2198
1412     17546
Name: Likes, Length: 1413, dtype: int64

In [27]:
px.histogram(df['Likes'],nbins=500)

In [28]:
px.line(df,y='Likes',x='Date',title='Number of likes per year',hover_data=['Title'])

We can see that number of likes increased as time passes on, but there are rapid jumps in line pattern. There are some videos with lot of likes and some videos have very few likes.

Lets see the videos which has lot of likes

In [29]:
df.sort_values('Likes',ascending=False)[:20]

Title     Views       Date  \
191                 Reflecting on the Color of My Skin   6648699 2020-06-04   
139              PlayStation 5 Unboxing & Accessories!  13652604 2020-10-27   
386                       Talking Tech with Elon Musk!  14229066 2018-08-17   
33                                       Dear YouTube!   3682313 2021-11-11   
144      iPhone 12 Unboxing Experience + MagSafe Demo!  12546306 2020-10-20   
124         AirPods Max Unboxing & Impressions: $550?!  11954658 2020-12-10   
385                 Tesla Factory Tour with Elon Musk!  13401942 2018-08-20   
219  Xiaomi Mi Mix Alpha Impressions: The Wraparoun...  15600590 2020-02-28   
351                   The Problem with YouTube Rewind!   7712667 2018-12-12   
316             Samsung Galaxy Fold Unboxing: Magnets!  23009027 2019-04-16   
216         The Truth About the Escobar Folding Phone!   8698647 2020-03-10   
267  This Is What Happens When You Re-Upload a YouT...   8158388 2019-10-11   
315                The Broken Galaxy Folds: Explained!  16611058 2019-04-18   
183                   iOS 14 Hands-On: Everything New!  10411900 2020-06-23   
242                                        10,000,000!   2944284 2019-12-18   
140                   iPhone 12 Review: Just Got Real!  12172825 2020-10-25   
154  Can You Actually Game in 8K? (RTX 3090 Gameplay!)   9787930 2020-09-22   
335   Talking Tech & Saving the World with Bill Gates!   5805260 2019-02-12   
43                        Reviewing EVERY iPhone Ever!   8502428 2021-10-11   
153        The Ultimate iOS 14 Homescreen Setup Guide!  11013258 2020-09-24   

      Likes  Comments                             Link  
191  657022     50188  youtube.com/watch?v=o-_WXXVye3Y  
139  580105     40834  youtube.com/watch?v=QtMzV73NAgk  
386  531345     27850  youtube.com/watch?v=MevKTPN4ozw  
33   510942     38042  youtube.com/watch?v=CaaJyRvvaq8  
144  431428     34317  youtube.com/watch?v=Sx6dAx7dnXg  
124  405425     46641  youtube.com/watch?v=UdfSrJvqY_E  
385  391474     19384  youtube.com/watch?v=mr9kK0_7x08  
219  390327     23535  youtube.com/watch?v=PTpoj4f25dk  
351  373757     35309  youtube.com/watch?v=P5dxd-ocaE8  
316  351793     11685  youtube.com/watch?v=x4yF3a3Zn4Y  
216  350032     18255  youtube.com/watch?v=O8FJSjy3bXA  
267  337055     19644  youtube.com/watch?v=JR4KHfqw-oE  
315  336113     30847  youtube.com/watch?v=vtqtyyGZvXM  
183  320297     25558  youtube.com/watch?v=ZLyDvABxGF0  
242  311607     20370  youtube.com/watch?v=NvQmi_ciL1k  
140  309617     21464  youtube.com/watch?v=X1b3C2081-Q  
154  304213     26754  youtube.com/watch?v=kFz9afj8lu0  
335  299530     18496  youtube.com/watch?v=4mxXdCUXSSs  
43   299491     20112  youtube.com/watch?v=8FpPSMIB4uA  
153  297461     13410  youtube.com/watch?v=cH66LWWluVE

## Lets focus on comments column

In [30]:
px.line(df,y='Comments',x='Date',title='Number of comments per year',hover_data=['Title'])

In [31]:
df.sort_values('Comments',ascending=False)[:20]

Title     Views       Date  \
365              RED iPhone XR Unboxing & Giveaway!   2415152 2018-10-23   
630                  Pick Your Smartphone Giveaway!    719215 2015-12-20   
680        Iron Man Edition Samsung Galaxy S6 Edge!   2037763 2015-06-05   
553     Dope Tech: Self Lacing Nike HyperAdapt 1.0!   1608385 2016-12-01   
118          Giving Away 24 PS5s and XBox Series X!   2197970 2020-12-25   
671                         iPod Touch 2015 Review!   2095810 2015-07-17   
647                         Update 11.0: MKBHD.com!    596689 2015-10-21   
191              Reflecting on the Color of My Skin   6648699 2020-06-04   
124      AirPods Max Unboxing & Impressions: $550?!  11954658 2020-12-10   
125             The Portless iPhone 13: Let's Talk!   6369708 2020-12-07   
139           PlayStation 5 Unboxing & Accessories!  13652604 2020-10-27   
452                  The $1000 Smartphone Giveaway!   1641292 2017-11-27   
152        The Galaxy S20 Fan Edition: Hear Me Out!   4869727 2020-09-28   
33                                    Dear YouTube!   3682313 2021-11-11   
1409  HP Pavilion dv7t Media Center Remote Overview   4640300 2009-01-01   
306                      The Huawei Ban: Explained!   7528004 2019-05-21   
255     Tesla CyberTruck Impressions & First Drive!  10447383 2019-11-22   
181             On the iPhone 12 With No Charger...   4470358 2020-07-01   
351                The Problem with YouTube Rewind!   7712667 2018-12-12   
150          Why Don't People Buy Sony Smartphones?   6930487 2020-10-05   

       Likes  Comments                             Link  
365   137624    339547  youtube.com/watch?v=vc1NdYXlaxA  
630    98466    242703  youtube.com/watch?v=ShcnNkr_PEY  
680    94709    177679  youtube.com/watch?v=VgmyVHYV1mE  
553    87264    105599  youtube.com/watch?v=C9Plog9I1JY  
118   203821     98975  youtube.com/watch?v=Lt-fah4spJY  
671    69410     68198  youtube.com/watch?v=orPHNS-0CZI  
647    25840     66130  youtube.com/watch?v=qv-RVkpQTWs  
191   657022     50188  youtube.com/watch?v=o-_WXXVye3Y  
124   405425     46641  youtube.com/watch?v=UdfSrJvqY_E  
125   230768     44570  youtube.com/watch?v=Qfmeb2e_kb4  
139   580105     40834  youtube.com/watch?v=QtMzV73NAgk  
452   152587     40391  youtube.com/watch?v=Wzvr_rtI82w  
152   153485     38704  youtube.com/watch?v=azrdcp4yYas  
33    510942     38042  youtube.com/watch?v=CaaJyRvvaq8  
1409  234631     37125  youtube.com/watch?v=9gk_rl3y_SU  
306   203392     36502  youtube.com/watch?v=qZGpmWrVSaU  
255   225399     35488  youtube.com/watch?v=oTZ84U-K_5k  
181   193678     35439  youtube.com/watch?v=8IB7JcKJEeI  
351   373757     35309  youtube.com/watch?v=P5dxd-ocaE8  
150   294507     34837  youtube.com/watch?v=j7QwJ-M_GAI

The videos with more number of comments are giveaway videos mostly.
And some other videos include discussion about some problems like 351.Youtube rewind, 150.Sony smartphones, 125.Portless iphone13, 33.Dear Youtube, 191.Reflecting on the color of my skin.

## Lets focus on Views column
### Main aspect of youtube

In [32]:
px.line(df,y='Views',x='Date',title='Number of views per year',hover_data=['Title'])

We can see the same pattern as for the likes

In [33]:
df.sort_values('Views',ascending=False)[:20]

Title     Views       Date  \
314                               Retro Tech: Game Boy  34282288 2019-04-19   
316             Samsung Galaxy Fold Unboxing: Magnets!  23009027 2019-04-16   
407              OnePlus 6 Review: Right On the Money!  21826363 2018-05-25   
462                           Apple iPhone X Unboxing!  17324220 2017-10-31   
315                The Broken Galaxy Folds: Explained!  16611058 2019-04-18   
219  Xiaomi Mi Mix Alpha Impressions: The Wraparoun...  15600590 2020-02-28   
386                       Talking Tech with Elon Musk!  14229066 2018-08-17   
139              PlayStation 5 Unboxing & Accessories!  13652604 2020-10-27   
385                 Tesla Factory Tour with Elon Musk!  13401942 2018-08-20   
144      iPhone 12 Unboxing Experience + MagSafe Demo!  12546306 2020-10-20   
140                   iPhone 12 Review: Just Got Real!  12172825 2020-10-25   
124         AirPods Max Unboxing & Impressions: $550?!  11954658 2020-12-10   
153        The Ultimate iOS 14 Homescreen Setup Guide!  11013258 2020-09-24   
64   Tesla Model S PLAID Impressions: Re-Inventing ...  10898289 2021-07-22   
577       iPhone 7 Unboxing: Jet Black vs Matte Black!  10869598 2016-09-14   
331                    Samsung Galaxy S10 Impressions!  10745547 2019-02-20   
263                AirPods Pro Unboxing & Impressions!  10520236 2019-10-29   
255        Tesla CyberTruck Impressions & First Drive!  10447383 2019-11-22   
183                   iOS 14 Hands-On: Everything New!  10411900 2020-06-23   
212         2020 iPad Pro Review: It's... A Computer?!  10170910 2020-03-24   

      Likes  Comments                             Link  
314  202456     14128  youtube.com/watch?v=Oy8zSYKkczI  
316  351793     11685  youtube.com/watch?v=x4yF3a3Zn4Y  
407  111404      9493  youtube.com/watch?v=0PrUr3bQdwM  
462  295967     23880  youtube.com/watch?v=l0DoQYGZt8M  
315  336113     30847  youtube.com/watch?v=vtqtyyGZvXM  
219  390327     23535  youtube.com/watch?v=PTpoj4f25dk  
386  531345     27850  youtube.com/watch?v=MevKTPN4ozw  
139  580105     40834  youtube.com/watch?v=QtMzV73NAgk  
385  391474     19384  youtube.com/watch?v=mr9kK0_7x08  
144  431428     34317  youtube.com/watch?v=Sx6dAx7dnXg  
140  309617     21464  youtube.com/watch?v=X1b3C2081-Q  
124  405425     46641  youtube.com/watch?v=UdfSrJvqY_E  
153  297461     13410  youtube.com/watch?v=cH66LWWluVE  
64   279913     18997  youtube.com/watch?v=DyKQ7qtTJag  
577  121065     10468  youtube.com/watch?v=J5HtSy5bATk  
331  231592     25033  youtube.com/watch?v=t9R7xx0joOU  
263  268137     18926  youtube.com/watch?v=MrUhzYdcX6w  
255  225399     35488  youtube.com/watch?v=oTZ84U-K_5k  
183  320297     25558  youtube.com/watch?v=ZLyDvABxGF0  
212  243682     13241  youtube.com/watch?v=x_R-qzjZrKQ

## Lets focus on Title column

In [34]:
df['Title']

0                            YouTube Needs to Fix This
1                   The Electric Car Pre-Order Problem
2       Does Fast Charging ACTUALLY Ruin Your Battery?
3             "The Apple Studio Display is a Bad Deal"
4                   Mac Studio Review: Double Trouble!
                             ...                      
1408                   Fraps HD Test in 1080p (18 WOS)
1409     HP Pavilion dv7t Media Center Remote Overview
1410                       High fps LG Voyager footage
1411             14 Year knock-down shot (11 Handicap)
1412                   13-Year-Old Golf Swing Analysis
Name: Title, Length: 1413, dtype: object

In [35]:
keywords=[]
for title in df['Title']:
    for word in title.split():
        keywords.append(word.lower())
d=Counter(keywords)
df_keys=pd.DataFrame({'Keys':d.keys(),'Count':d.values()})

In [36]:
df_keys.sort_values('Count',ascending=False)[:30]

Keys  Count
5              the    250
972             hd    161
1858     tutorial:    134
52          galaxy    119
26         review:    118
673        review!    104
35          iphone     87
219        samsung     85
135         google     83
269              5     80
430          mkbhd     72
205   impressions!     70
268            top     66
80             pro     62
22               a     61
88      explained!     60
63              of     58
70    impressions:     51
1805          [hd]     51
0          youtube     50
1288         nexus     48
90            tech     46
146             in     44
40             new     43
42               &     41
2               to     40
85              vs     39
78         oneplus     39
687            ask     36
18           apple     35

In the start, he used phrase 'hd tutorial', so it is in top in frequent number of words

Some of the other most used keywords are galaxy, review, iphone, samsung, google, mkbhd, impressions, youtube

### Lets find the videos with apple, iphone, samsung, galaxy

In [37]:
df_keys[(df_keys['Keys']=='apple')|(df_keys['Keys']=='iphone')|(df_keys['Keys']=='samsung')|(df_keys['Keys']=='galaxy')]

Keys  Count
18     apple     35
35    iphone     87
52    galaxy    119
219  samsung     85

In [38]:
t=df_keys[(df_keys['Keys']=='apple')|(df_keys['Keys']=='iphone')|(df_keys['Keys']=='samsung')|(df_keys['Keys']=='galaxy')]
px.bar(x='Keys',y='Count',data_frame=t)

There are more number of videos of samsung when compared to apple, as samsung releases more mobiles than apple

Lets count number of unboxing videos

In [39]:
df[df['Title'].str.lower().str.contains('unboxing')].sort_values('Views',ascending=False)

Title     Views       Date  \
316              Samsung Galaxy Fold Unboxing: Magnets!  23009027 2019-04-16   
462                            Apple iPhone X Unboxing!  17324220 2017-10-31   
139               PlayStation 5 Unboxing & Accessories!  13652604 2020-10-27   
144       iPhone 12 Unboxing Experience + MagSafe Demo!  12546306 2020-10-20   
124          AirPods Max Unboxing & Impressions: $550?!  11954658 2020-12-10   
577        iPhone 7 Unboxing: Jet Black vs Matte Black!  10869598 2016-09-14   
263                 AirPods Pro Unboxing & Impressions!  10520236 2019-10-29   
83                       Apple AirTags Unboxing & Demo!   8440312 2021-04-22   
49                    iPhone 13 Unboxing & Impressions!   8218958 2021-09-21   
287                  Apple Card Unboxing & Impressions!   7903810 2019-08-09   
376                        Gold iPhone Xs Max Unboxing!   4976914 2018-09-18   
246   Mac Pro Unboxing & Second Impressions: The Pow...   4936076 2019-12-10   
276                     Unboxing Every Green iPhone 11!   4586858 2019-09-17   
40                 Pixel 6/6 Pro Unboxing & First Look!   4439220 2021-10-19   
223   Samsung Galaxy Z Flip Unboxing: It's Growing o...   4437999 2020-02-19   
526                          New RED iPhone 7 Unboxing!   4123172 2017-03-21   
472                 LG 77" Wallpaper TV Unboxing [LIVE]   3569435 2017-09-25   
162   Samsung Galaxy Z Fold 2 Unboxing: 3 Major Upgr...   3468824 2020-09-02   
163    Surface Duo Unboxing & Impressions: Hinge Goals!   3034100 2020-08-26   
568   Google Pixel Unboxing! (Best Android 7.1 Featu...   2964069 2016-10-18   
473                  iPhone 8 Unboxing: Silver vs Gold!   2760180 2017-09-21   
381        RED Hydrogen One Unboxing! (Houdini Edition)   2684761 2018-08-31   
420                      Product RED iPhone 8 Unboxing!   2652477 2018-04-10   
655          iPhone 6s & 6s Plus Unboxing & First Look!   2643965 2015-09-22   
466   Google Pixel 2 Unboxing! (Best Android 8.0 Fea...   2599986 2017-10-17   
187                  The Largest Daily Driver Unboxing!   2591801 2020-06-16   
563         13" MacBook Pro 2016 Unboxing! (What's New)   2518470 2016-10-28   
228      Foldable Moto RAZR Unboxing & Second Thoughts!   2443295 2020-02-10   
365                  RED iPhone XR Unboxing & Giveaway!   2415152 2018-10-23   
570                  Dope Tech: 8K RED Epic-W Unboxing!   2276122 2016-10-11   
556           LG Ultrafine 5K Display Unboxing + Setup!   1465064 2016-11-23   
648                          Nexus 6P Unboxing & Setup!   1385718 2015-10-19   
665                         OnePlus 2 Unboxing & Setup!   1144394 2015-08-11   
726                 Google Nexus 6 Unboxing + Hands On!   1010613 2014-11-04   
800                 New Mac Pro Unboxing & Impressions!    695875 2014-01-13   
725                 Google Nexus 9 Unboxing + Hands On!    587506 2014-11-05   
888      Chromebook Pixel Unboxing & First Impressions!    187957 2013-02-25   
921              Nexus 10 Unboxing & First Impressions!    176930 2012-11-16   
919               Nexus 4 Unboxing & First Impressions!    150208 2012-11-20   
1072                       Canon EOS Rebel T2i Unboxing     45771 2010-11-25   
1096        Dell Ultrasharp 2407WFPb Unboxing and Setup     43359 2010-08-16   
1010        Verizon Galaxy Nexus Unboxing & First Boot!     43025 2011-12-15   
1095                      Cyborg R.A.T. 7 Unboxing [HD]     37229 2010-08-26   
1113                 HTC Droid Incredible Unboxing [HD]     25564 2010-04-30   
1101                              Droid X Unboxing [HD]     24055 2010-07-20   
1124                         DealExtreme Unboxing! [HD]     22844 2010-02-10   
1219                            ThinkGeek Unboxing [HD]     17580 2009-05-25   
1129                       Gateway HX2000 Unboxing [HD]     14779 2010-01-11   
1132                                   iBellix Unboxing      7291 2009-12-27   
1159                            iSkin Surprise Unb

Above are 51 unboxing videos in the channel sorted according to popularity(Views)

Most viewed unboxings are
1. Samsung galaxy fold
2. iphone x
3. Playstation 5
4. iphone 12
5. Airpods max
6. Iphone 7

There are 3 iphones in top 6

Lets perform the same for review videos

In [40]:
df[df['Title'].str.lower().str.contains('review')].sort_values('Views',ascending=False)

Title     Views       Date  \
407              OnePlus 6 Review: Right On the Money!  21826363 2018-05-25   
140                   iPhone 12 Review: Just Got Real!  12172825 2020-10-25   
212         2020 iPad Pro Review: It's... A Computer?!  10170910 2020-03-24   
830                            Apple iPhone 5s Review!   9971750 2013-10-11   
204           iPhone SE (2020) Review: It All Adds Up!   9563276 2020-04-22   
...                                                ...       ...        ...   
1206  17" Laptop GelaSkin Installation and Review [HD]      6296 2009-06-12   
1074                          Review: AeroWeather [HD]      4861 2010-11-21   
1320                         HD Skype Review (Request)      3571 2009-02-17   
1214      Reviewed: Solo 17" Laptop Messenger Bag [HD]      2774 2009-06-01   
1340                              HD Review: Camstudio      2277 2009-02-10   

       Likes  Comments                             Link  
407   111404      9493  youtube.com/watch?v=0PrUr3bQdwM  
140   309617     21464  youtube.com/watch?v=X1b3C2081-Q  
212   243682     13241  youtube.com/watch?v=x_R-qzjZrKQ  
830   138077     20038  youtube.com/watch?v=uBEaG6n0XDs  
204   296734     26852  youtube.com/watch?v=aXfiyuUziY0  
...      ...       ...                              ...  
1206      76        20  youtube.com/watch?v=ClRH1aQuP_Q  
1074      69        28  youtube.com/watch?v=a0qY7W-9nbE  
1320      68        14  youtube.com/watch?v=mhhgh84ZDmE  
1214      44        20  youtube.com/watch?v=XV_WP85r4MY  
1340      38        14  youtube.com/watch?v=PkRXVGJ0ods  

[271 rows x 6 columns]

Most viewed reviews are

1. Oneplus 6
2. Iphone 12
3. 2020 ipad pro
4. iphone 5s
5. iphone se 2020

There are 4 apple devices in top 4

There are 271 review videos. 

# Insights

1. He released a lot of videos in 2009
2. He released only 3 videos in 2008 which were 3,5,9 months of the year. Each video in each month.
3. In 2009 feb alone, he relesed 93 videos. That's a lot when compared to other months in that year.
4. Not much insights gained from likes, comments, views column. Just found videos with highest in each category. The things I observed is they are not consistent, some videos here and there had lot of likes, views. Of course it happens as not every video will not get hit.
5. Most viewed unboxings are
    1. Samsung galaxy fold
    2. iphone x
    3. Playstation 5
    4. iphone 12
    5. Airpods max
    6. Iphone 7

    There are 3 iphones in top 6.
6. Most viewed reviews are

    1. Oneplus 6
    2. Iphone 12
    3. 2020 ipad pro
    4. iphone 5s
    5. iphone se 2020

    There are 4 apple devices in top 5.
    Oneplus 6 in the top😍. I miss that old oneplus😔.